automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/FreeTrialSpec.scala

```diff
@@ -66,7 +66,10 @@ class FreeTrialSpec extends FreeSpec with BeforeAndAfterEach with Matchers with
         setUpEnabledUserAndProject(testUser)
         val trialAuthToken = TrialBillingAccountAuthToken()
         api.trial.reportTrialProjects().foreach { x =>
-          register cleanUp Rawls.admin.deleteBillingProject(x.name, UserInfo(OAuth2BearerToken(trialAuthToken.value), WorkbenchUserId("0"), WorkbenchEmail("doesnt@matter.com"), 3600))(UserPool.chooseAdmin.makeAuthToken())
+          register cleanUp Try(Rawls.admin.deleteBillingProject(x.name, UserInfo(OAuth2BearerToken(trialAuthToken.value),
+            WorkbenchUserId("0"), WorkbenchEmail("doesnt@matter.com"), 3600))(UserPool.chooseAdmin.makeAuthToken())).recover {
+              case _: RestException => logger.warn(s"Error occurred in Rawls.admin.deleteBillingProject(${x.name})")
```

Comment:
warn if GPAllocator delete billing project throws `"message":"Sam call to HttpMethod(DELETE) /api/resource/billing-project/gpalloc-qa-master-4savc2m failed"`